manifests/init.pp

```diff
@@ -990,7 +990,7 @@
             require => Package[$my_package_name],
           }
 
-          file { 'pam_common_noninteractive_session':
+          file { 'pam_common_session_noninteractive':
```

Comment:
I guess I don't understand what reversing the name buys us. Does it make spec tests easier or something?